CalcualtorLogic.js

```diff
@@ -119,7 +119,7 @@ class Calculator {
         const clkedOperButn = event.target.dataset.operator;
         this.operator = clkedOperButn;
         this.lastButtonClicked = "opertaor"
-        this.lastValue = this.resultElement;
+        this.updateResultElement();
```

Comment:
What is the reason of calling `updateResultElement` here?
what do you want to do?

Reply:
where are you saving the current result in another property?

Reply:
is the clear button working now?

Reply:
yes It is 
thanks !!